packages/app-project/components/ClassifyPage/components/ClassifierWrapper/ClassifierWrapperContainer.js

```diff
@@ -51,6 +60,14 @@ export default class ClassifierWrapperContainer extends Component {
       )
     }
 
+    if (project.loadingState === asyncStates.error) {
+      return (
+        <p>
```

Comment:
This could reuse the `ErrorMessage` component - https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/components/ClassifyPage/components/ClassifierWrapper/components/ErrorMessage/ErrorMessage.js

Reply:
Good point.

Reply:
I've added a simple test which should display project loading errors or classifier errors with the `ErrorMessage` component, I think.